pylama.ini

```diff
@@ -0,0 +1,6 @@
+[pylama]
```

Comment:
Let's either completely remove this config if we're removing Pylama, or add it to `setup.cfg` and remove this file.